File: C#/02. Simple Operations and Calculations/Exercise/Dance Hall.cs
using System;

namespace Dance_Hall
{
    class Program
    {
        static void Main(string[] args)
        {
            double L = double.Parse(Console.ReadLine()) * 100;
            double W = double.Parse(Console.ReadLine()) * 100;
            double A = double.Parse(Console.ReadLine()) * 100;

            double hallArea = L * W ;
            double benchArea = hallArea / 10;
            double wardrobeArea = A * A ;

            hallArea -= (benchArea + wardrobeArea);
            Console.WriteLine(Math.Floor(hallArea / (40 + 7000)));
        }
    }
}

File: Python/02. Simple Operations and Calculations/Exercise/Dance Hall.py
import math

L = float(input()) * 100
W = float(input()) * 100
A = float(input()) * 100

danceHallArea = L * W
wardrobeArea = A * A
benchArea = danceHallArea / 10
dancerSpace = 40 + 7000
dancers = (danceHallArea - wardrobeArea - benchArea) / dancerSpace

print(math.floor(dancers))